packages/trip-details/src/trip-detail.tsx

```diff
@@ -41,19 +41,25 @@ export default class TripDetail extends Component<Props, State> {
     const { icon, summary, description } = this.props;
     const { expanded } = this.state;
     return (
-      <S.TripDetail>
-        <S.TripDetailIcon>{icon}</S.TripDetailIcon>
+      <S.TripDetail role="group">
+        <S.TripDetailIcon role="presentation">{icon}</S.TripDetailIcon>
         <S.TripDetailSummary>
           {summary}
           {description && (
-            <S.ExpandButton onClick={this.toggle}>
+            <S.ExpandButton id="expand-button" onClick={this.toggle}>
```

Comment:
Could we use a `<summary>`/`<details>` duo instead?

Reply:
Add `aria-expanded`, and `aria-controls` to the button. Move `aria-label` to this level and support i18n.

Reply:
Is it details though? I wouldn't say that the "summary" summarizes what's inside the expansion box

Reply:
The `<summary>`/`<details>` combo could be added in at a later time, but that's not needed for this PR.

Reply:
I didn't anticipate the id generation haha!

Reply:
😭